assets/js/app.js

```diff
@@ -6,13 +6,15 @@ import {
   Link,
   NavLink,
 } from 'react-router-dom';
+import PrivateRoute from './component/security/PricateRoute';
 import ReactGA from 'react-ga';
 import Index from './container/index/index';
 import UserNav from './component/user/user-nav';
 import ThingListContainer from './container/thing/list';
 import ThingDetailContainer from './container/thing/detail';
 import RequestPasswordResetModal from './component/modal/request-password-reset';
 import logo from '../logo-print4health-org.svg';
+import Dashbord from './container/dashboard/dashbord';
```

Comment:
minor typo - should be "Dashboard"
```suggestion
import Dashboard from './container/dashboard/dashboard';
```